packages/ckeditor5-widget/src/widgetresize/resizerstate.ts

```diff
@@ -189,9 +189,11 @@ export default class ResizeState extends ObservableMixin() {
 }
 
 /**
- * Calculates a relative width of a `domResizeHost` compared to its ancestor in percents.
+ * Calculates a relative width of a `domResizeHost` compared to its ancestor in pixels.
+ *
+ * @internal
  */
-function calculateHostPercentageWidth( domResizeHost: HTMLElement, resizeHostRect: Rect ) {
+export function calculateHostWidth( domResizeHost: HTMLElement ): number {
```

Comment:
I don't think we'll get away with making this internal and exporting in the package. If exported, this will have to have proper API docs and unit tests.

Reply:
@oleq API docs updated, unit tests added.